Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -319,6 +319,30 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.9,
 )
 
+# MLPF workflows
+class UpgradeWorkflow_mlpf(UpgradeWorkflow):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        if 'Reco' in step:
+            stepDict[stepName][k] = merge([self.step3, stepDict[step][k]])
+    def condition(self, fragment, stepList, key, hasHarvest):
+        return fragment=="TTbar_14TeV" and '2026' in key
```

Comment:
is it worthwhile to enable this test workflow for Run2/3 also?

Reply:
Yes, this is a good idea and could be considered. Now I wanted to introduce just a single workflow to make sure it runs.

Reply:
I switched it to 2021 now, as we the training for Phase2 is still to be completed & studied.